src/components/tab-view/__tests__/TabViewPrimitive-test.js

```diff
@@ -393,11 +393,33 @@ describe('components/tab-view/TabViewPrimitive', () => {
                 });
             });
 
-            describe('UNSAFE_componentWillReceiveProps', () => {
+            describe('componentDidUpdate', () => {
+                // test('should not call focusOnTabElment if component is not dynamic', () => {
```

Comment:
Is this no longer needed?

Reply:
No it is, just haven't updated it yet. The PR is still a draft, so some of it is midstream.